lib/top_row.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_projects/save_movie_button.dart';
+import 'package:flutter_projects/stars.dart';
+
+import 'like_counter.dart';
+
+import 'movie.dart';
+
+/*
+* Represents the row at the top right corner, between the background and the poster image.
+* It works as a container for the save button, like button, and the rating stats.
+* */
+class TopRow extends StatelessWidget {
+  const TopRow({
+    super.key,
+    required this.movie,
+  });
+
+  static const double posterContainerWidth = 170;
+  static const double posterContainerHeight = 230;
+  static const double buttonsTopPadding = 80;
+  static const double buttonsLeftPadding = 15;
+  final Movie movie;
+
+  static const BoxShadow posterBoxShadow = BoxShadow(
+    blurRadius: 18,
+    spreadRadius: 0.75,
+    //color:,Theme.of(context).colorScheme.inversePrimary,
+    offset: Offset.zero, blurStyle: BlurStyle.normal,
+  );
+
+  static const voteAverageTextStyle = TextStyle(
+    fontSize: 15,
+    fontWeight: FontWeight.bold,
+    color: Colors.grey,
+  );
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Container(
+          width: posterContainerWidth,
+          height: posterContainerHeight,
+          decoration: BoxDecoration(
+            boxShadow: const [posterBoxShadow],
+            image: DecorationImage(
+              image: movie.poster.image,
+            ),
+          ),
+        ),
+        Padding(
+          padding: const EdgeInsets.only(
+            top: buttonsTopPadding,
+            left: buttonsLeftPadding,
+          ),
+          child: Row(
+            children: [
+              const SaveMovieButton(),
+              LikeCounter(movie),
+              Column(
+                children: [
+                  Text(movie.voteAverage.toString(),
+                      style: voteAverageTextStyle),
+                  Stars(movie),
+                ],
+              ),
+            ],
+          ),
+        ),
+      ],
+      //children
```

Comment:
Please delete this comment, as it's not needed

Reply:
done